pkg/proxy/apis/config/validation/validation_test.go

```diff
@@ -354,15 +326,17 @@ func TestValidateKubeProxyConfiguration(t *testing.T) {
 					TCPCloseWaitTimeout:   &metav1.Duration{Duration: 5 * time.Second},
 				},
 			},
-			msg: "only one CIDR allowed or a valid DualStack CIDR (e.g. 10.100.0.0/16,fde4:8dba:82e1::/48)",
+			//TODO(cmluciano): Consider revising this error to mention that we are requiring 1 valid IPv6 CIDR or
+			// that mixing two IPfamilies is not permitted
```

Comment:
I'm not sure that leaving TODO comments in code is a good way to leave notes to the future. If you really think this should be done then file an issue about it. (Unless you're just leaving a note for yourself because you plan to do further PRs here...)

(If you do leave the comment, you need a space after the `//` on the first line.)

Reply:
SGTM, this is indeed something that I intend to fix later. I will open an issue to correspond with this too.